docs/content/troubleshooting/troubleshoot-support.md

```diff
@@ -0,0 +1,45 @@
+---
+docs: DOCS-1648
+doctypes:
+- troubleshooting
+draft: false
+tags:
+- docs
+title: Support
+toc: true
+weight: 100
+---
+
+F5 NGINX Ingress Controller adheres to the support policy detailed in the following knowledge base article: [K000140156](https://my.f5.com/manage/s/article/K000140156).
+
+In order to open a support ticket, F5 would like additional information to better understand the problem.
+
+The [nginx-supportpkg-for-k8s](https://github.com/nginxinc/nginx-supportpkg-for-k8s) plugin collects the information needed by F5 Technical Support to assist with troubleshooting your issue.
+
+The plugin leverages [krew](https://krew.sigs.k8s.io), the plugin manager for the Kubernetes [kubectl](https://kubernetes.io/docs/reference/kubectl/) command-line tool.
+
+The plugin may collect some or all of the following global and namespace-specific information:
+
+* k8s version, nodes information, and CRDs
+* Pods' logs
+* list of Pods, events, ConfigMaps, Services, Deployments, StatefulSets, ReplicaSets, and Leases
+* k8s metrics
+* helm Deployments
+* `nginx -T` output from NGINX-related pods
+
+This plugin **DOES NOT** collect secrets or coredumps.
```

Comment:
```suggestion
This plugin **does not** collect secrets or coredumps.
```

Bold is already used for emphasis: if it requires further attention, a warning callout should be used instead of caps.